content/release-notes/2022-09-01.md

```diff
@@ -5,6 +5,5 @@ label: 'Storage'
 ### What's new
 
 - Compute: Updated the PostgreSQL version to 14.5.
-- Compute: Added support for the `PostGIS` extension, version 3.3.0.
-- Pageserver: Changed basebackup import panics to plain errors.
+- Compute: Added support for the `PostGIS` extension, version 3.3.0. Supported extensions can be installing using `CREATE EXTENSION` syntax. For information about PostgreSQL extensions supported by Neon, see [Available PostgreSQL extensions](/docs/reference/compatibility/#available-postgresql-extensions).
```

Comment:
Haiving the link to the extensions page is useful, not sure about repeating the docs in the release notes.